torch/__init__.py

```diff
@@ -1266,7 +1266,8 @@ def _check_tensor_all(cond, message=None):  # noqa: F811
 # For Python Array API (https://data-apis.org/array-api/latest/API_specification/constants.html) and
 # NumPy consistency (https://numpy.org/devdocs/reference/constants.html)
 from math import e , nan , inf , pi
-__all__.extend(['e', 'pi', 'nan', 'inf'])
+newaxis: None = None
```

Comment:
The type of none is `NoneType`.

Also, no need to keep the history clean, as we squeeze all the commits on merge

Reply:
nvm
>Note that None as a type hint is a special case and is replaced by type(None).

Reply:
> Also, no need to keep the history clean, as we squeeze all the commits on merge

Thanks, I did not know :pray: